docs/UserGuide.md

```diff
@@ -147,66 +148,74 @@ Examples:
 
 `personfind Nicole` finds all the employees named ‘Nicole’
 
-`personfind * Hardware` finds all employees tagged with ‘Hardware’
-
 `personfind Nicole Hardware Display` finds all employees named ‘Nicole’, tagged with BOTH 'Hardware' AND 'Display'
 
-### List all persons : `personlist`
+### List all persons : `list`
 Lists all the employees in the company.
 
-Format: `personlist`
+Format: `list`
 
-Example: `personlist` shows all the employees in the company
+Example: `list` shows all the employees in the company
 
 ### Clear all entries : `clear`
 
 Removes all the employees’ information in the company from the app. Useful for clearing out sample data.<br>
 ❗️Caution: You cannot recover the data afterwards.
 
-[coming in v1.3] Require confirmation with a `yes clear` for clearing all data with `clear`.
-Format: `clear`
-
 Example: `clear` removes all the employees and jobs from the app
 
+
+### Export a person's information : `export`
+
+Export a `.csv` file with the person's name as file name, containing the jobs the contractor worked on, and the hours
+they worked for, and how much pay they should expect to receive this month.
+
+This helps contractors check if their hours and pay is accurately reflected. This increases the transparency of the SME
+they are working for.
+
+Format: `export PERSON_INDEX`
+
+Example: `export 3` exports the third person in the list
+
 --------------------------------------------------------------------------------------------------------------------
 
 ## Job-related commands
 
-### Add a job : `add`
+### Add a job : `job`
 
 Adds a new job to the system with the given attributes. `RATE` refers to how much the employee is paid per hour. `DURATION` refers to how long the job took.
 
-Format: `add n/NAME d/DURATION`
+Format: `job n/NAME d/DURATION`
 
-Example: `add 2 n/Fix HDB Lock d/1` creates a job with id 2, where the employees worked for 1 hour to fix a HDB lock
+Example: `job 2 n/Fix HDB Lock d/1` creates a job with id 2, where the employees worked for 1 hour to fix a HDB lock
 
-## Search for a job by name : `find`
+## Search for a job by name : `jobfind`
```

Comment:
The current CRUD commands for jobs are the shortened versions. Did you rebase from master? `add` adds a job now, and `personadd` adds a person.